content/examples/podcasts.md

```diff
@@ -0,0 +1,124 @@
+---
+title: "Sustain Open Source Design Podcast"
+icon: "ti-microphone"
+description: "All about meaningful contributions between Open Source and Design"
+type: "docs"
+---
+
+S.O.S. Design is a podcast dedicated to exploring the intersection of open source and design: how design is crucial in the open source ecosystem, how designers work with coders to make open source software better, and what sustainability means for the field of open source designers. This podcast grew out of the Sustain community and Open Source Design, and seeks to share great conversations with members from both communities and the open source and design space at large.
+
+[Episode 2: Abraham Agiri and Designing as a Coder, and Coding as a Designer](https://sosdesign.sustainoss.org/2)
+
+- `[00:36:13]` Eriol’s spotlight is Penpot.
+- `[00:36:43]` Justin’s spotlight is Taiga6.
+- `[00:37:09]` Memo’s spotlight is Open Web Docs.
+- `[00:37:40]` Richard’s spotlight is NodeSchool.
+- `[00:38:04]` Agiri’s spotlight is NodeJs and OpenJs Foundation.
```

Comment:
Are the timestamps particularly useful here? I'm curious for thoughts from other reviewers.

Reply:
I think timestamps are always useful but I do wonder if they make sense to folks without context of 'Timestamp' or 'Skip to this time to hear Eriol's spotlight' etc. I think that might be my ALT text addition/change

Reply:
I think they can find the timestamps on the podcast show notes for that podcast. May not be useful to include here.

Reply:
Should we do some change here?

Reply:
I would probably remove them as they're elsewhere. @Erioldoesdesign What do you think?

Reply:
makes sense! what richard said!

Reply:
removed timestamps from episode list